.rubocop.yml

```diff
@@ -144,3 +144,7 @@ InternalAffairs/UndefinedConfig:
   Exclude:
     - 'lib/rubocop/cop/correctors/**/*.rb'
     - 'lib/rubocop/cop/mixin/**/*.rb'
+
+InternalAffairs/StyleDetectedApiUse:
+  Exclude:
+    - 'lib/rubocop/cop/mixin/percent_array.rb'
```

Comment:
`InternalAffairs/StyleDetectedApiUse` is registering a false positive here because it's detecting `no_acceptable_style` as a negative but the positive is within `array_style_detected`.

Interestingly, there does not seem to be any way to inline disable global offenses...